lib/main.ts

```diff
@@ -86,9 +104,23 @@ class AchYouDataProviderEngine
     }
     return [];
   };
+
+  /**
+   * @param selector string user id or general selector
+   */
+  getUser = async (selector: string) => {
+    if (!selector) return [];
+    else if (selector === '*') return this.users;
+    else if (typeof selector === 'string') {
+      return this.users.filter((user) => {
+        return selector === user.userId;
+      });
+    }
+    return [];
+  }
 }
 
-const sneezeBoardWebViewType = "sneeze-board.sneeze-board";
```

Comment:
I wonder if this line was accidentally deleted.

Reply:
Yes :)